channels/tj.m3u

```diff
@@ -1,3 +1,6 @@
 #EXTM3U
+
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="http://www.tamby.info/tv-tadjikistana-online/images/tvt.png" group-title="",Televizioni Tojikiston
+http://185.121.2.6:8081/live/tvt1214/chunks.m3u8
```

Comment:
I don't know exactly what the problem is, but I can only see a black screen from the link. Tested with VLC, Elmedia Video Player and streamtest.in